javascript/javascript1/week2/Event-applicatoin.js

```diff
@@ -1,2 +1,11 @@
-weekday [ 'monday', 'tuesday', 'wednesday', 'thursday',
-'friday', 'saturday', 'monday' ];
\ No newline at end of file
+const weekDays = ['monday', 'tuesday', 'thursday', 'wednesday',
+ 'thursday', 'friday', 'saturday', 'sunday'];
+const today = new Date();
+const day = today.getDay()
+let numDaysToEvent;
```

Comment:
As you used numDaysToEvent as parameter to the function, you need not initialise the same here.